bip-taro-addr.mediawiki

```diff
@@ -79,7 +111,7 @@ program, as specified in BIP 341, using the included key as the internal key.
 
 With the target taproot public key script constructed, the asset is sent to the
 receiver with the execution of the following steps:
-# Construct a valid transaction that spends an input that holds the referenced <code>asset_id</code> and ''at least'' <code>amt</code> units of the asset.
+# Construct a valid transaction that spends an input that holds the referenced <code>asset_id</code> and ''exactly'' <code>amt</code> units of the asset.
 # Create a new Taro output commitment based on the input commitment (this will be the change output), that now only commits to <code>S-A</code> units of <code>asset_id</code>, where <code>S</code> is the input amount, and <code>A</code> is the amount specified in the encoded Taro address.
 ## This new leaf MUST have a <code>split_commitment</code> specified that commits to the position (keyed by <code>sha256(output_index || asset_id || asset_script_key)</code> within the transaction of the newly created asset leaf for the receiver.
 ## Add an additional output that sends a de minimis (in practice this MUST be above dust) amount to the top-level taproot public key computed earlier.
```

Comment:
Would be clearer with variable name instead of 'key computed earlier'. Interpreted this as paying to the root that included the serialized TLV.